src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/TestScenario.cs

```diff
@@ -12,12 +15,15 @@ public class TestScenario
     public string ScenarioName { get; }
     public DotNetTemplate Template { get; }
 
-    public TestScenario(string scenarioName, DotNetLanguage language, DotNetTemplate template, DotNetActions commands = DotNetActions.None)
+    private Action<string>? Validate;
```

Comment:
Consider the following for consistency with the other properties.

```suggestion
    public Action<string>? Validate { get; }
```